src/macaron/output_reporter/templates/base_template.html

```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c) 2022 - 2022, Oracle and/or its affiliates. All rights reserved. -->
+<!-- Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. -->
```

Comment:
Copyright notice should keep the start year, i.e. should be`2022 - 2023`.

Reply:
Addressed in https://github.com/oracle/macaron/pull/165/commits/927bfe16e500d716e475ed7b04b33a7532fb98b8